packages/lib-classifier/src/plugins/drawingTools/models/tools/Tool/Tool.js

```diff
@@ -50,10 +50,18 @@ const Tool = types.model('Tool', {
       self.marks.delete(mark.id)
     }
 
+    function reset () {
+      self.marks.forEach((mark) => {
+        mark.reset()
```

Comment:
Won't `marks.clear()` on the next line automatically clear out `mark.annotations` for each mark?

Reply:
No I don't think so for the same reason that I gave for the ClassificationsStore, but this line isn't necessary since you've pointed out that the Mark model is a composition with the AnnotationsStore, so the observer can be modified to observe either parent regardless and reset itself if the parent resets itself.